site/get-started-developing.md

```diff
@@ -0,0 +1,154 @@
+# Get Started Developing Flux
+
+This tutorial is going to walk you through one possible workflow for making a small (actually, tiny) change to flux and building and testing that change locally.  Before we get started there are some baseline prerequisites you're gonna need:
+
+## Run `flux-getting-started`
+
+1. We're going to make some changes soon enough, but just to get a good baseline please follow the [Getting Started](site/get-started.md) guide and run the `flux-getting-started` repo through its normal paces.
+
+    Now that we know everything is working with `flux-getting-started`, we're going to try and do nearly the same thing as `flux-getting-started`, except instead of using official releases of flux, we're going to build and run what we have locally.
+
+## Prepare your Environment
+
+1. Install the prerequisites.  This guide is written from running Linux, but the same instructions will generally apply to OSX.  Although everything you need has been known to work independently in Windows from time to time, results may vary.
+    - [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    - [Docker](https://docs.docker.com/install/)
+    - [Go](https://golang.org/doc/install)
+
+1. We want to make sure we're starting fresh.  Tell Minikube to clear any previously running clusters.
+    ```sh
+    minikube delete
+    ```
+
+1. The [Minikube addon](https://github.com/kubernetes/minikube/blob/master/docs/addons.md) called [freshpod](https://github.com/GoogleCloudPlatform/freshpod) that will be very useful to us later.  You'll see.  It's gonna be cool.
+    ```sh
+    minikube addons enable freshpod
```

Comment:
Can you please replace Freshpod with [Skaffold](https://github.com/GoogleContainerTools/skaffold), Freshpod is no longer maintained.

Reply:
Yes.  @squaremo had suggested I used freshpod.  That, and it being super easy (with the one-liner) to integrate was the reason for using it.